### PR TITLE
Tag AWSCore et all

### PR DESCRIPTION
**Pull request by samoconnor**

### CODE REVIEW

File: InfoZIP/versions/0.0.10/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
+Compat
```
Comment (tkelman) [inline InfoZIP/versions/0.0.10/requires]: It's best to declare a minimum version dependency that implements the features you need.

Comment (samoconnor) [inline InfoZIP/versions/0.0.10/requires]: Fair enough...

But there is no change to `requires` vs 0.0.9: https://github.com/JuliaLang/METADATA.jl/blob/metadata-v2/InfoZIP/versions/0.0.9/requires

Is this a METADATA.jl-update blocking issue?

**Comment by tkelman** [comment]
Compat is missing from SymDict's REQUIRE, and it should not be modifying Base methods on Base types.

**Comment by samoconnor** [comment]
I've added Compat to SymDict's REQUIRE: https://github.com/samoconnor/SymDict.jl/blob/master/REQUIRE  See #8533

Re: modifying Base methods on Base types...
Is this a general comment on SymDict? or has this inadvertently changed since v0.0.5 ?

**Comment by samoconnor** [comment]
See #8533

**Comment by tkelman** [comment]
General comment, but was not noticed before. It's bad practice now and was bad practice before.

**Comment by samoconnor** [comment]
As far as I can tell, none of the `Base.merge` or `Base.get` methods in SymDict.jl are modifying base methods insofar as commenting out any of them causes the tests to throw a `MethodError`. i.e. they all seem to be new methods. Am I missing something?

**Comment by tkelman** [comment]
They're adding methods that are not present in Base, with signatures that consist only of Base types. That means unrelated code that may have been expecting an error, or using a different code path, can have its semantics subtly changed by importing your package because of the global modification of method tables you're doing.

**Comment by samoconnor** [comment]
Ok, so adding methods is bad practice in addition to modifying methods.
I've added a new issue to SymDict: https://github.com/samoconnor/SymDict.jl/issues/2

**Pull request closed without merging**